ipmininet/cli.py

```diff
@@ -81,6 +81,25 @@ def do_ping6pair(self, _line):
         """Ping (IPv6-only) between first two hosts, useful for testing."""
         self.mn.ping6Pair()
 
+    def do_linkFailure(self,line):
+        """Down the link between 2 nodes"""
+        nodes = line.split(' ')
+        if(len(nodes) != 2):
+            lg.output("Error: specify 2 nodes")
+        else:
+            self.mn.runFailurePlan([(nodes[0],nodes[1])])
+
+    def do_restoreLink(self,line):
```

Comment:
This limitation should not be there.

I'd suggest to keep track of the failed links in either the Net or CLI object (e.g., a dict[(src,dst)|src|dst]->Link), such that you could do:
linkFailure u v, v w, w x, y z
then
linkRestore v, w, y z
to restore all failed links from v/w and the y z link